components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -73,11 +72,39 @@
 {#if manifest && manifest.error}
   <nylas-domain-error {id} />
 {:else}
-  {#each editableProperties as prop}
+  <div>
     <label>
-      <strong>{prop.label}</strong>
-      <input value={prop.value} />
+      <strong>Event Title</strong>
+      <input type="text" bind:value={manifestProperties.event_title} />
     </label>
-  {/each}
+  </div>
+  <div>
+    <label>
+      <strong>Event Description</strong>
+      <input type="text" bind:value={manifestProperties.event_description} />
+    </label>
+  </div>
+  <div>
+    <strong>Show meeting hosts to the end-user?</strong>
+    <label>
+      <input
+        type="radio"
+        name="show_hosts"
+        bind:group={manifestProperties.show_hosts}
+        value="show"
+      />
+      <span>Show Hosts</span>
+    </label>
+    <label>
+      <input
+        type="radio"
+        name="show_hosts"
+        bind:group={manifestProperties.show_hosts}
+        value="hide"
+      />
+      <span>Hide Hosts</span>
+    </label>
+  </div>
```

Comment:
This is a good candidate for a fieldset element as there are two inputs.
```
<fieldset>
  <legend>Show meeting hosts to the end-user?</legend>
  <label><input /></label>
  <label><input /></label>
 <fieldset>   
```

Reply:
Noting for posterity: @heathergh and I came to the conclusion to do this for fieldset/legend-having-implicit-styles reasons:

```
  <div role="radiogroup" aria-labelledby="show_hosts">
    <strong id="show_hosts">Show meeting hosts to the end-user?</strong>
```

See: https://www.w3.org/TR/WCAG20-TECHS/ARIA17.html#ARIA17-examples